README.md

```diff
@@ -171,6 +171,18 @@ Dexter.withActivity(activity)
 	.check();
 ```
 
+### Force
+For some reason, you may want to force permission asking even if Android system is 
+telling that permission is granted. 
```

Comment:
Do you have an example of this? I'm trying to get my head around but I cannot find any scenario where this should happen.